package.json

```diff
@@ -12,7 +12,8 @@
     "@emotion/react": "11.7.1",
     "@emotion/styled": "11.6.0",
     "@mui/icons-material": "5.3.1",
-    "@mui/material": "5.3.1",
+    "@mui/material": "^5.4.1",
+    "@mui/styles": "^5.4.1",
```

Comment:
ATTENTION!!! Never never never use the `^` symbol in this project in package versions. It's the reason of conflicts in yarn.lock file which bright the big problems for us lately